### ISSUE TITLE
Crash more gracefully when using --dt

### ISSUE THREAD

**Issue by DanielRosenwasser** [Microsoft/dts-gen#57]
While dts-gen shouldn't crash, it's just going to sometimes. That's fine, but DefinitelyTyped used to have a script that consistently worked. Now the instructions are to rely on dts-gen entirely, and having just tried it on a library (`moonjs`), it just crashed.

This package needs to have some sort of general graceful fallback behavior if DefinitelyTyped is going to rely on it. Otherwise we're making it way too annoying/difficult to contribute.

CC @andy-ms @RyanCavanaugh @mhegazy

**Comment by mhegazy** [comment]
Makes sense. If it failed it should at least create the file with the right header.
